runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeConfig.java

```diff
@@ -123,4 +123,14 @@ default boolean showOresFound()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "showLootAsItems",
+		name = "Toggle item icons",
+		description = "Shows the gems/ores as their items rather than text"
+	)
+	default boolean showLootAsItems()
```

Comment:
Change this and the keyName to `showLootIcons()` or something similar

Reply:
Change made